docs/develop/dapps/payment-processing/accept-payments-in-a-telegram-bot-js.md

```diff
@@ -0,0 +1,440 @@
+# Bot for the sale of dumplings
+
+
+In this article, we'll create a simple Telegram bot for accepting payments in TON coins.
+
+## 🦄 What it looks like
+
+The bot will look like this:
+
+![bot preview](https://raw.githubusercontent.com/ton-community/ton-docs/main/static/img/tutorials/js-bot-preview.png?raw=true)
+
+### Source code
+
+Sources are available on GitHub:
+* https://github.com/coalus/DumplingShopBot
+
+## 📖 What you'll learn
+You'll learn how to:
+ - Create a Telegram bot in NodeJS using Grammy
```

Comment:
We need to cover more about Grammy and why it's necessary. I don't know what is it, and overview here would be pretty helpful